src/main/java/com/zufar/onlinestore/payment/service/PaymentFailedScenarioHandler.java

```diff
@@ -0,0 +1,28 @@
+package com.zufar.onlinestore.payment.service;
+
+import com.stripe.model.PaymentIntent;
+import com.zufar.onlinestore.payment.entity.Payment;
+import com.zufar.onlinestore.payment.enums.PaymentStatus;
+import com.zufar.onlinestore.payment.repository.PaymentRepository;
+import lombok.AllArgsConstructor;
+import lombok.NoArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Service;
+
+@Slf4j
+@AllArgsConstructor
+@NoArgsConstructor
+@Service
+public class PaymentFailedScenarioHandler implements PaymentScenarioHandler {
```

Comment:
add java doc please
it helps to understand this class for people who have no idea about stripe